### DISCUSSION TITLE
一台远程收割机，能收割多少块田，有没限制或者效率问题

### DISCUSSION THREAD

**Discussion by wanming851** [Chia-Network/chia-blockchain#6698] [Support]
一台远程收割机，能收割多少块田，有没限制或者效率问题

**Comment by cccat6** [comment]
这取决于你收割机的配置以及挂载方案。官方建议（不记得在哪看到了）是单机500T差不多了。我个人实际挂到2P以后开始出现比较明显的性能问题。我是用的PC挂，如果你用树莓派什么的应该挂的更少。具体的值必须你自己测试，这东西没有标准答案。
另外建议你删掉另一条提问，机翻的全是错误，完全看不懂在说啥。

**Comment by BrandtH22** [comment]
@wanming851 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
